x-pack/test/functional/page_objects/monitoring_page.ts

```diff
@@ -10,20 +10,11 @@ import { FtrProviderContext } from '../ftr_provider_context';
 export function MonitoringPageProvider({ getPageObjects, getService }: FtrProviderContext) {
   const PageObjects = getPageObjects(['common', 'header', 'security', 'login']);
   const testSubjects = getService('testSubjects');
-  const security = getService('security');
-
   return new (class MonitoringPage {
     async navigateTo(useSuperUser = false) {
-      // always create this because our tear down tries to delete it
-      await security.user.create('basic_monitoring_user', {
-        password: 'monitoring_user_password',
-        roles: ['monitoring_user', 'kibana_admin'],
-        full_name: 'basic monitoring',
-      });
-
       if (!useSuperUser) {
```

Comment:
Ideally , we wouldn't need to do this - because when we restore defaults, it is the `superuser` . But good to check this .- Can get tricky if other tests are using other `test_user` logins.